common/src/Admin.php

```diff
@@ -69,6 +70,13 @@ public function options_page() {
             'invite_delay' => 'intval',
             'limit_order_data' => 'intval',
             'javascript' => 'boolval',
+            'order_statuses' => function ($value) {
```

Comment:
Don't know what is the correct way to sanitize this array. 🤦

Reply:
`function ($value) { return array_map('strval', is_array($value) ? $value : []); }`